src/VrRetreat.WebApp/Controllers/AccountController.cs

```diff
@@ -108,4 +108,100 @@ public async Task<IActionResult> Logout()
         await _signInManager.SignOutAsync();
         return RedirectToAction(nameof(HomeController.Index), "Home");
     }
+
+    [HttpGet]
+    public IActionResult Settings()
```

Comment:
Let's add an `[Authorize]` attribute to make sure only logged in users can view the page and you can then skip the `IsSignedIn` check. 

You see, that check is done in routes where you are always allowed to view it, but it might do something else if you're logged in. (for example the index) ☺️